subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -1261,7 +1272,8 @@ SHELL_STATIC_SUBCMD_SET_CREATE(wifi_commands,
 		    "[-b, --bands <Comma separated list of band values (2/5/6)>] : Bands to be scanned where 2: 2.4 GHz, 5: 5 GHz, 6: 6 GHz.\n"
 		    "[-a, --dwell_time_active <val_in_ms>] : Active scan dwell time (in ms) on a channel. Range 5 - 1000 ms.\n"
 		    "[-p, --dwell_time_passive <val_in_ms>] : Passive scan dwell time (in ms) on a channel. Range 10 - 1000 ms.\n"
-		    "[-s, --ssid <Comma separate list of SSIDs>] : SSID list to scan for.",
+		    "[-s, --ssid <Comma separate list of SSIDs>] : SSID list to scan for.\n"
+		    "[-m, --max_bss <val>] : Max BSSIDs to scan for. Range 1 - 65535.",
```

Comment:
0 - 65535 (though 0 is default, we can still include it in the range)

Reply:
This is intentional since default is 0 (for unlimited results) and user doesnt need to specify this option if he wants unlimited results.